src/network/netdev/fou-tunnel.c

```diff
@@ -47,6 +47,10 @@ static int netdev_fill_fou_tunnel_message(NetDev *netdev, sd_netlink_message **r
         if (r < 0)
                 return log_netdev_error_errno(netdev, r, "Could not append FOU_ATTR_PORT attribute: %m");
 
+        r = sd_netlink_message_append_u16(m, FOU_ATTR_PEER_PORT, htobe16(t->peer_port));
+        if (r < 0)
+                return log_netdev_error_errno(netdev, r, "Could not append FOU_ATTR_PEER_PORT attribute: %m");
```

Comment:
From the code in `netdev_fou_tunnel_verify()`, I guess that `peer_port` should be sent only when peer address is set. If that is true, then please do so.